doc/conf.py

```diff
@@ -3,42 +3,65 @@
 import sys, os
 sys.path.insert(0, os.getcwd())
 
-from builder.shared_conf import * # noqa (API import)
-
-paths = ['../param/', '.', '..']
-add_paths(paths)
-
-from ..setup import setup_args
+from nbsite.shared_conf import *
 
 # Declare information specific to this project.
 project = u'HoloViews'
 authors = u'PyViz developers'
-copyright = u'2018 ' + authors
-ioam_module = 'holoviews'
+copyright = u'2019 ' + authors
+pyviz_module = 'holoviews'
 description = 'Stop plotting your data - annotate your data and let it visualize itself.'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = setup_args['version']
+version = 'v1.11.0' #setup_args['version']
 # The full version, including alpha/beta/rc tags.
-release = setup_args['version']
+release = 'v1.11.0' #setup_args['version']
 
 ASSETS_URL = 'https://assets.holoviews.org'
 
 rst_epilog = """
 .. _tutorial notebooks: {url}/notebooks-{version}.zip
 """.format(url=ASSETS_URL, version=version)
 
-# Override IOAM theme
-html_theme = 'holoviews_theme'
+extensions += ['nbsite.gallery']
+
+nbsite_gallery_conf = {
+    'backends': ['bokeh', 'matplotlib', 'plotly'],
+    'galleries': {
+        'gallery': {
+            'title': 'Gallery',
+            'sections': [
+                {'path': 'apps', 'title': 'Applications', 'skip': True},
+                'demos'
+            ]
+        },
+        'reference': {
+            'path': 'reference',
+            'sections': [
+                'elements',
+                'containers',
+                'streams',
+                'apps'
+            ],
+            'title': 'Reference Gallery',
+        }
+    },
+    'github_org': 'pyviz',
+    'github_project': 'holoviews'
```

Comment:
could these be a level higher and be used by generate-rst and build as well?

Reply:
Probably.

Reply:
Ok that would be a good goal for nbsite. Then we could just write a `doit build_docs` in pyctdev and wouldn't need to run `nbsite generate-rst` and `nbsite build` with all the flags everytime.

Reply:
Should handle this in nbsite and then update it here. Will merge this as is for now.